vault/resource_generic_secret_test.go

```diff
@@ -115,6 +176,98 @@ func testResourceGenericSecret_initialCheck(expectedPath string) resource.TestCh
 	}
 }
 
+func testResourceGenericSecret_initialCheck_v2(expectedPath string, wantValue string, wantVersion int) resource.TestCheckFunc {
```

Comment:
Here is it is `wantVersion` and below it is `expectedVersions`, maybe rename to `versionCount` or `wantNumVersions` for both funcs.

Reply:
Fixed in d9476a4